cluster-operator/src/main/java/io/strimzi/operator/cluster/operator/resource/StatefulSetOperator.java

```diff
@@ -347,4 +280,15 @@ protected StatefulSetDiff revertStorageChanges(StatefulSet current, StatefulSet
 
         return new StatefulSetDiff(current, desired);
     }
+
+    protected Future<Integer> getGen(String namespace, String podName) {
```

Comment:
I think we should avoid abbreviations in names as much as possible, so this should be `getGeneration`, or even just `generation`